### PR TITLE
Updating docs with loopback info

### PR DESCRIPTION
**Pull request by DeTeam**
- Added missing info to `proxy/doc.go`
- Fixed a few typos

### CODE REVIEW

File: proxy/doc.go [hunk 1]
```diff
@@ -62,6 +63,13 @@ If a filter chain was broken by some filter this step is skipped.
 In case the route is a 'shunt', an empty response is created with
 default 404 status.
 
+3.c loopback:
+
+In case of the `loopback` route no upsteram request is made.
+Routing will happen again with an augmented request.
+This can happen sevearl times in a row until maxLoopbacks
```
Comment (aryszka) [inline proxy/doc.go]: sevearl -> several

Comment (DeTeam) [inline proxy/doc.go]: Thanks, already spotted + a few more in other docs.

**Comment by DeTeam** [comment]
👍

**Comment by aryszka** [comment]
:+1:

**Pull request merged**